libs/core/src/lib/wizard/wizard-content/wizard-content.component.html

```diff
@@ -0,0 +1,12 @@
+<ng-template #contentTemplate>
+    <section
+        class="fd-wizard__content"
+        [ngClass]="
+            (contentBackground ? 'fd-wizard__content--' + contentBackground : '') +
+            ' ' +
+            (size ? 'fd-wizard__content--' + size : '')
+        "
```

Comment:
Can we move this logic to a getter at least? Shall it be convenient to use array notation for that?

Reply:
I think it is cleaner to use this than to add an additional getter function that will just do the same thing